learning_journal_basic1/views/default.py

```diff
@@ -0,0 +1,98 @@
+from pyramid.response import Response
+from pyramid.view import view_config
+from sqlalchemy.exc import DBAPIError
+
+from pyramid.httpexceptions import HTTPFound
+from datetime import datetime
+
+from ..models import MyModel
+
+TIME_FORMAT = '%B %d, %Y'
+
+
+@view_config(route_name='home', renderer='../templates/list.jinja2')
+def home_view(request):
+    """Display a list of all entries in the database on Home Page."""
+    try:
+        query = request.dbsession.query(MyModel)
+        entries = query.all()
+    except DBAPIError:
+        return Response(db_err_msg, content_type='text/plain', status=500)
```

Comment:
I think it's probably better to remove the above two lines entirely.  You'll still get a good traceback when developing locally, and in production you won't leak potentially sensitive information about the structure of your app.